scripts/lsync.sh

```diff
@@ -3,29 +3,68 @@
 # This script checks if the English version of a page has changed since a localized
 # page has been committed.
 
-if [ "$#" -ne 1 ] || ! [ -f "$1" ]; then
-  echo -e "\nThis script checks if the English version of a page has changed since a "
-  echo -e "localized page has been committed.\n"
+if [ "$#" -lt 1 ]; then
+  echo -e "\nThis script checks if the English version of a page has changed since a " >&2
+  echo -e "localized page has been committed.\n" >&2
   echo -e "Usage:\n\t$0 <PATH>\n" >&2
   echo -e "Example:\n\t$0 content/zh/docs/concepts/_index.md\n" >&2
   exit 1
 fi
 
-LOCALIZED="$1"
-
-# Try get the English version
-EN_VERSION=`echo $LOCALIZED | sed "s/content\/..\//content\/en\//g"`
-if ! [ -f $EN_VERSION ]; then
-  echo "$EN_VERSION has been removed."
+# Validate paths
+VALID=1
+for ARG in "$@"
+do
+  if printf "%s" "$ARG" | grep -E "^/|^\.\./|^\.\.\$|/\.\./|/\.\.\$"; then
+    printf "%s: Invalid path \"%s\"\n" "$0" "$ARG" >&2
+    VALID=0
+  fi
+done
+if [ $VALID = 0 ]; then
   exit 2
 fi
 
-# Last commit for the localized file
-LASTCOMMIT=`git log -n 1 --pretty=format:%h -- $LOCALIZED`
+EXITSTATUS=0 # assume success
+for LOCALIZED in "$@"
+do
+
+  # Try get the English version
+  EN_VERSION="$( printf "%s" "$LOCALIZED" | sed "s/content\/..\//content\/en\//g" )"
```

Comment:
this line doesn't work?

Reply:
See https://github.com/kubernetes/website/pull/24419#discussion_r501386957

If you're seeing an error, let me know what it looks like.